src/main/java/com/icthh/xm/uaa/config/UaaAccessTokenConverterConfiguration.java

```diff
@@ -104,15 +108,17 @@ private static PublicKey getKeyFromConfigServer(RestTemplate keyUriRestTemplate)
     }
 
     private PrivateKey getPrivateKey() throws IOException, KeyStoreException, CertificateException,
-        NoSuchAlgorithmException, UnrecoverableKeyException {
-        log.info("Keystore location {}", applicationProperties.getKeystoreFile());
-        try (InputStream stream = new ClassPathResource(applicationProperties.getKeystoreFile()).exists()
-            ? new ClassPathResource(applicationProperties.getKeystoreFile()).getInputStream()
-            : new FileInputStream(new File(applicationProperties.getKeystoreFile()))) {
-            KeyStore store = KeyStore.getInstance(Constants.KEYSTORE_TYPE);
-            store.load(stream, applicationProperties.getKeystorePassword().toCharArray());
-            return (PrivateKey) store.getKey("selfsigned",
-                                             applicationProperties.getKeystorePassword().toCharArray());
+        NoSuchAlgorithmException, UnrecoverableKeyException, InvalidKeySpecException {
+        if (!org.springframework.util.StringUtils.isEmpty(applicationProperties.getPrivateKey())) {
+            log.info("Keystore loaded from memory");
```

Comment:
suggest changing log message to : `Key was loaded from memory by application property: app_property_name`